packages/web/utils/string.ts

```diff
@@ -5,3 +5,17 @@ export function truncateString(str: string, num = 8) {
   }
   return str.slice(0, num) + "...";
 }
+
+export function getShortAddress(
+  address: string,
+  opts?: { prefixLength?: number; suffixLength?: number }
```

Comment:
FYI we do have this utility. It struggles at too low of a cutoff, but works nevertheless.

https://github.com/osmosis-labs/osmosis-frontend/blob/28d515b21ea8a999496790d7e82bf29ab90b0115/packages/web/components/complex/transfer.tsx#L123

Reply:
Replaced in favor of existing utility, thanks 👍